src/aleph/vm/orchestrator/resources.py

```diff
@@ -122,6 +123,21 @@ async def about_system_usage(_: web.Request):
     return web.json_response(text=usage.json(exclude_none=True))
 
 
+@cors_allow_all
+async def about_certificates(request: web.Request):
+    """Public endpoint to expose platform certificates for confidential computing."""
+
+    if not settings.ENABLE_CONFIDENTIAL_COMPUTING:
+        return web.HTTPBadRequest(reason="Confidential computing setting not enabled on that server")
+
+    sev_client: SevClient = request.app["sev_client"]
+
+    if not sev_client.certificates_archive.is_file():
+        await sev_client.export_certificates()
+
+    return web.FileResponse(sev_client.certificates_archive)
```

Comment:
Would it not be preferable to move the caching logic inside the `SevClient` class ?

```suggestion
    return web.FileResponse(await sev_client.get_certificates())
```